OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -395,4 +395,13 @@ public interface IRemoveFrozenActor
 
 	public interface IRulesetLoaded<TInfo> { void RulesetLoaded(Ruleset rules, TInfo info); }
 	public interface IRulesetLoaded : IRulesetLoaded<ActorInfo>, ITraitInfoInterface { }
+
+    public interface IResource
+    {
+        string ResourceName { get; }
```

Comment:
If we're going to start explicitly implementing interfaces everywhere then these names are more verbose than they need to be.